server/drivers/hd44780.c

```diff
@@ -238,18 +238,13 @@ static int parse_backlight_option( const char *value )
 	return -1;
 }
 
-static const char *backlight_type_str(int backlight_type)
+/* reports textually setting of backlight */
+static void report_backlight_type(int report_level, int backlight_type)
 {
-	static char buffer[256] = "";
+	/* should be enough for current options set and all possible combinations */
+	char buffer[256] = "";
```

Comment:
While this is certainly true, it is very easy to miss, when adding new options. So at least we should check against overflow in the code below.